pom.xml

```diff
@@ -296,5 +303,26 @@
 				</pluginManagement>
 			</build>
 		</profile>
+		<profile>
+			<id>verify-iplog</id>
```

Comment:
Maybe add a doc for how to generate the report, and mention that generating the report requires Java 11.

Reply:
I've made a proposed addition to the README.